eng/tools/generator/typespec/typespecgo/option.go

```diff
@@ -0,0 +1,18 @@
+package typespecgo
+
+/*
+https://github.com/Azure/autorest.go/blob/main/packages/typespec-go/src/lib.ts#GoEmitterOptions
+@azure-tools/typespec-go option
+*/
+type GoEmitterOptions struct {
+	AzcoreVersion         string `yaml:"azcore-version"`
```

Comment:
Seems this type has difference from the definition in `typespec-go` emitter? Does this include all the options the user can set in `tspconfig.yaml`?

Reply:
GoEmitterOptions is not actually used, it is just a reference value here, remove it?

Reply:
yes, let's remove it.